packages/dev-tools/src/actions/WriteSummary.tsx

```diff
@@ -56,6 +56,11 @@ export function WriteSummary({ write }: Props) {
           .filter(isDefined)
       : null;
 
+  const functionSelectorAndArgs: Hex = write.request?.args?.length
```

Comment:
I think this won't always be the case if a system is called with arguments. Unfortunately the template doesn't have a good example of this but I think the example/minimal does (look for `InventorySystem.sol`).

Instead we might have to check if the request method matches `call` or `callFrom` and then decode accordingly.